frontends/PyCDE/test/polynomial.py

```diff
@@ -22,17 +22,18 @@ class PolynomialCompute:
 
   def __init__(self, coefficients: list[int], **kwargs):
     """coefficients is in 'd' -> 'a' order."""
-    self.__coefficients = coefficients
+    self.coefficients = Parameter(coefficients)
     # Full result.
     self.y = Output(types.i32)
 
   @generator
-  def construct(mod):
+  def construct(mod, params):
```

Comment:
No action item, but eventually we should document this signature somewhere (once the API has stabilized).

Reply:
Yeah, right now it's mlir Attributes, which are pretty worthless IME.